omod/src/main/webapp/fragments/field/personRelationship.gsp

```diff
@@ -16,11 +16,11 @@
             <select id="relationship_type" name="relationship_type" class="rel_type" ng-model="relationship.type">
                 <option value="">${ui.message('registrationapp.person.relationship.selectRelationshipType')}</option>
                 <% relationshipTypes.each { type -> %>
-                <option value="${type.uuid}-A">${ui.message(type.aIsToB)}</option>
+                <option value="${type.uuid}-A">${ui.encodeHtmlContent(ui.message(type.aIsToB))}</option>
                 <% } %>
                 <% relationshipTypes.each { type -> %>
                 <% if (type.aIsToB != type.bIsToA) { %>
-                <option value="${type.uuid}-B">${ui.message(type.bIsToA)}</option>
+                <option value="${type.uuid}-B">${ui.message(ui.encodeHtmlContent(type.bIsToA))}</option>
```

Comment:
thanks @Parth59  have you tested on a running instance of openmrs?

Reply:
Yes yes. I recently tried running it on the https://demo.openmrs.org/openmrs/registrationapp/registerPatient.page?appId=referenceapplication.registrationapp.registerPatient and was able to spot the vulnerability, 
After making the change I am able to see that the vulnerability is fixed, Is it that the PR code is breaking down at your end ?